metrics/github/transformers/github_wrapper.sh

```diff
@@ -0,0 +1,27 @@
+# Wrapper for running the main github transformer
+
+set -eux
+# Install other tools
+apt update && apt install -y jq
+
+# Install python tools we need
+pip3 install --upgrade \
+  awscli \
+  poetry \
+
+# get the repository we need and set it up
+git clone --depth 1 https://github.com/mozilla-services/GitHub-Audit
+cd GitHub-Audit
+poetry install
+
+# create credential files
+echo -e "\n$githubAPItoken" >./.credentials
+mkdir -p ~/.aws
+cat >~/.aws/credentials <<EOF
+[cloudservices-aws-stage]
+aws_access_key_id = $AWS_ACCESS_KEY_ID
```

Comment:
You are 😉  , there's the disclaimer above:
> The "download" code is in a separate repository, intended for standalone use -- this code only includes and executes that repo. Due to the need to run independently, some of the credential storage is based on that. While the docker container does have credentials written to files, the container is removed at the end of each run.

The standalone query uses the "standard" ``~/.aws/credentials`` approach, so that's why I stuff them into that format within the container. I think that's a reasonable thing for the wrapper to handle.

Reply:
ok, in that case I'm good with that :)